lib/stoplight/version.rb

```diff
@@ -1,5 +1,5 @@
 # coding: utf-8
 
 module Stoplight
-  VERSION = Gem::Version.new('1.0.0')
+  VERSION = Gem::Version.new('1.1.0')
```

Comment:
I will handle bumping the version number when I do a release, like 10eae8e6f09c5fa28ff9b5903e380ef7b4e83ef9.